pkgs/top-level/all-packages.nix

```diff
@@ -24846,7 +24846,9 @@ with pkgs;
 
   logiops = callPackage ../misc/drivers/logiops { };
 
-  logitech-udev-rules = callPackage ../os-specific/linux/logitech-udev-rules { };
+  # ltunifi and solaar both provide udev rules but solaar's rules are more
+  # up-to-date so we simply use that instead of having to maintain our own rules
+  logitech-udev-rules = solaar.udev;
```

Comment:
This should go into `pkgs/top-level/aliases.nix`

Reply:
> This should go into `pkgs/top-level/aliases.nix`

...only if `logitech-udev-rules` is a backwards compatibility alias, right? I interpret it as a persistent alias, not something that should be removed, so I think the current location is fine. I guess @oxalica can clarify.

Reply:
> > This should go into `pkgs/top-level/aliases.nix`
> 
> ...only if `logitech-udev-rules` is a backwards compatibility alias, right? I interpret it as a persistent alias, not something that should be removed, so I think the current location is fine. I guess @oxalica can clarify.

+1. I think `aliases.nix` is for deprecated names or persistent renaming. Just like why we not putting `rust = rust_1_61;
` in it.

Reply:
I don't care about this point enough, happy with whatever you choose.